update.go

```diff
@@ -57,6 +68,30 @@ other options are ignored.
 			Name:  "blkio-weight",
 			Usage: "Specifies per cgroup weight, range is from 10 to 1000",
 		},
+		cli.IntFlag{
+			Name:  "blkio-leaf-weight",
+			Usage: "Specifies tasks' weight in the given cgroup while competing with the cgroup's child cgroups, range is from 10 to 1000, cfq scheduler only",
```

Comment:
That's not what I meant. What I meant was that the *actual range* should not be in the help docs. The ranges are defined by the `runtime-spec`, not by us.